__tests__/app.test.js

```diff
@@ -66,4 +65,62 @@ describe("GET/api", () => {
                 expect(endpointObject["GET /api/articles"].exampleResponse).not.toBeUndefined();
             });
     });
+})
+
+//retrieving by their id number
+describe("GET /api/articles/:article_id", () => {
+    test("200: when retrieving an article by an id number of 1, return the correct article information as an object", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body;
+            const expectedArticle = {
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 100,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              }
+            expect(article).toEqual(expectedArticle)
+        })
+    })
+    test("200: when retrieving an article by a valid id number (but this time more large), return the correct article information as an object", () => {
+        return request(app)
+        .get("/api/articles/12")
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body;
+            const expectedArticle = {
+                article_id: 12,
+                title: "Moustache",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "Have you seen the size of that thing?",
+                created_at: "2020-10-11T11:24:00.000Z",
+                votes: 0,
+                article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"}
+            expect(article).toEqual(expectedArticle)
+        })
+    })
+    test("404: when given an article_id number which is not in the database, return an error message", () => {
+        return request(app)
+        .get("/api/articles/65")
+        .expect(404)
+        .then(({ body }) => {
+            expect(body.msg).toBe('Invalid get request: ID not found');
```

Comment:
This is a great test, but I would amend the language from _"invalid"_ to just _"ID not found"_ 
( invalid data would pertain to a `400` instead, in this case it is valid but just doesn't hold any data )